tooltester/src/main/resources/JenkinsReadme.md

```diff
@@ -5,7 +5,7 @@ Last tested with ansible 2.2.1.0
 3.  Create a pipeline called PipelineTest
 4.  Copy the contents of resources/PipelineTest.groovy into the Pipeline Script textbox
 5.  Check the checkbox:  "This project is parameterized"
-6.  Create 5 String Parameters with the names: "URL", "Tag", "DescriptorPath", "ParameterPath", "DockerfilePath"
+6.  Create the String Parameters mentioned in the constructParameterMap(...) function
```

Comment:
Maybe link to the source file, also explain what this step is doing

Reply:
Will change it in my other PR since this function is not in develop yet (so it's more difficult to link to it).